microsoft-edge/web-platform/json-viewer.md

```diff
@@ -0,0 +1,86 @@
+---
+title: View a JSON file as-is or formatted
+description: View a formatted or minified JSON file as-is or formatted, in Microsoft Edge.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 03/05/2025
+---
+# View a JSON file as-is or formatted
+
+When you open a JSON file in Microsoft Edge, the file is initially displayed as-is (nested, or minified), and the **Pretty-print** checkbox is initially cleared.
+
+
+A `.json` file that contains whitespace and nesting ([128KB.json](https://microsoftedge.github.io/Demos/json-dummy-data/128KB.json)), displayed as-is:
+
+![A .json file that's non-minified, initially opened in Edge](./json-viewer-images/non-minified-json-file-as-is.png)
```

Comment:
I think we need to remove these 2 lines. If the JSON file is already nicely formatted, there's no need for the tool. The user will already be able to easily view the content.

Reply:
removed png & 
```
A `.json` file that contains whitespace and nesting ([128KB.json](https://microsoftedge.github.io/Demos/json-dummy-data/128KB.json)), displayed as-is:

![A .json file that's non-minified, initially opened in Edge](./json-viewer-images/non-minified-json-file-as-is.png)
```